### ISSUE TITLE
Kicad DRC gives me 357 hole clearence violations.

### ISSUE THREAD

**Issue by mrkskk** [anarion80/porcupine#4]
Runnng DRC before generating gerbers gives me a lot of errors.
So im questioning whether the PCB will still work?

**Comment by anarion80** [comment]
Yes, this is because the reversible, hot-swap choc switch footprint has the hot-swap pads overlapping a bit with the holes for switch. You should downgrade that error to violation. Anyway, this is ok and the PCB will work.

**Comment by mrkskk** [comment]
Sounds good. sending it unedited to JLC now 👍
